content/influxdb/v2.0/write-data/best-practices/schema-design.md

```diff
@@ -0,0 +1,229 @@
+---
+title: InfluxDB schema design
+description: >
+  Improve InfluxDB schema design and data layout. Store unique values in fields and other tips to reduce high cardinality in InfluxDB and make your data more performant.
+menu:
+  influxdb_2_0:
+    name: Schema design
+    weight: 201
+    parent: write-best-practices
+---
+
+Each InfluxDB use case is unique and your [schema](/influxdb/v2.0/reference/glossary/#schema) design reflects that uniqueness. Discover a few design guidelines that we recommend for most use cases:
+
+- [Where to store data (tags or fields)](#where-to-store-data-tags-or-fields)
+- [Avoid too many series](#avoid-too-many-series)
+- [Use recommended naming conventions](#use-recommended-naming-conventions)
+<!-- - [Recommendations for managing shard group duration](#shard-group-duration-management)
+-->
+
+{{% note %}}
+Follow these guidelines to minimize high series cardinality and make your data more performant.
+{{% /note %}}
+
+## Where to store data (tags or fields)
+
+[Tags](/influxdb/v2.0/reference/glossary/#tag) are indexed and [fields](/influxdb/v2.0/reference/glossary/#field) are not.
+This means that queries on tags are more performant than queries on fields.
+
+In general, your queries should guide what gets stored as a tag and what gets stored as a field:
+
+- Store commonly-queried meta data in tags.
+- Store data in fields if each data point contains a different value.
+- Store numeric values as fields ([tag values](/influxdb/v2.0/reference/glossary/#tag-value) only support string values).
+
+## Avoid too many series
+
+[Tags](/influxdb/v2.0/reference/glossary/#tag) containing highly variable information like UUIDs, hashes, and random strings lead to a large number of [series](/influxdb/v2.0/reference/glossary/#series) in the database, also known as high [series cardinality](/influxdb/v2.0/reference/glossary/#series-cardinality).
+
+High series cardinality is a primary driver of high memory usage for many database workloads.
+When you write to InfluxDB, InfluxDB uses the measurements and tags to create indexes to speed up reads.
+
+However, when there are too many indexes created, both writes and reads may start to slow down. Therefore, if a system has memory constraints, consider storing high-cardinality data as a field rather than a tag.
+
+{{% note %}}
+If reads and writes to InfluxDB have started to slow down, you may already have high series cardinality (too many series). See how to [resolve high cardinality](/influxdb/v2.0/write-data/best-practices/resolve-high-cardinality/).
+{{% /note %}}
```

Comment:
If reads and writes to InfluxDB ~have started~ start to slow down, you may ~already~ have high series cardinality (too many series)...